src/components/childComponents/Keypad.js

```diff
@@ -1,39 +1,179 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-function keypad() {
+function Keypad({ handleClick /* handleClear, handleCalculate  */ }) {
   return (
+
     <div className="keypad">
       <div className="row">
-        <button type="button" className="key">AC</button>
-        <button type="button" className="key">+/-</button>
-        <button type="button" className="key">%</button>
-        <button type="button" className="key operator">/</button>
+        <button
+          type="button"
+          name="AC"
+          className="key"
+          onClick={() => handleClick('AC')}
+        >
+          AC
+        </button>
+        <button
+          type="button"
+          name="+/-"
+          className="key"
+          onClick={() => handleClick('+/-')}
+        >
+          +/-
+        </button>
+        <button
+          type="button"
+          name="%"
+          className="key"
+          onClick={() => handleClick('%')}
+        >
+          %
+        </button>
+        <button
+          type="button"
+          name="+"
+          className="key operator"
+          onClick={() => handleClick('/')}
+        >
+          /
+        </button>
```

Comment:
- [ ] When I use this to calculate, it throws error and it is because the expected parameter is wrong, it should be `%`. Kindly fix it.